scripts/zowe-parse-yaml.sh

```diff
@@ -203,6 +203,21 @@ do
                 ZOWE_SERVER_PROCLIB_MEMBER=$value
                 export ZOWE_SERVER_PROCLIB_MEMBER
             fi
+            if [[ $key == "keystoreDirectory" ]] && [[ $section == "keystore" ]]
+            then
+                KEYSTORE_DIRECTORY=$value
+                export KEYSTORE_DIRECTORY
+            fi
+            if [[ $key == "keystoreOwner" ]] && [[ $section == "keystore" ]]
+            then
+                ZOWE_USER_ID=$value
+                export ZOWE_USER_ID
+            fi
+            if [[ $key == "keystoreGroup" ]] && [[ $section == "keystore" ]]
+            then
+                ZOWE_GROUP_ID=$value
```

Comment:
Can you add these new parameters to the echo sections below that handle the defaults if missing and logging the property values please. The tests are failing and without that info I can't see all that is going on. Thanks!